modules/parent-join/src/internalClusterTest/java/org/elasticsearch/join/aggregations/ChildrenIT.java

```diff
@@ -173,7 +173,7 @@ public void testWithDeletes() throws Exception {
             assertThat(children.getDocCount(), equalTo(4L));
 
             Sum count = children.getAggregations().get("counts");
-            assertThat(count.getValue(), equalTo(4.));
+            assertThat(count.value(), equalTo(4.));
```

Comment:
The `getValue` method was on the `Sum` interface and duplicated the `value` method on the internal classes.